tensorboard/plugins/audio/tf_audio_dashboard/tf-audio-dashboard.html

```diff
@@ -160,6 +160,10 @@ <h3>No audio data was found.</h3>
         });
       },
 
+      _shouldOpen(index) {
+        return index <= 2;
```

Comment:
nit: I think this should be 1, right?

Reply:
Surprisingly enough, the indexes appear to start at 1.

Reply:
Ah, thanks for the info!

Reply:
The indexes actually start at 0: https://stackoverflow.com/questions/31388072/polymer-1-0-dom-repeat-display-index-starting-at-1

The reason that <= 2 works is because the 0th pane is the search pane, apparently:
https://github.com/tensorflow/tensorboard/blob/master/tensorboard/components/tf_categorization_utils/categorizationUtils.ts#L104

Reply:
Ah, that makes sense!